docs/roadmap.md

```diff
@@ -95,3 +85,7 @@ This isn't an exhaustive list: it's a sampling of some of the key infrastructure
   
 * Moving engineering to GitHub
   * [Proposals](https://github.com/microsoft/ProjectReunion/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature+proposal%22) -> [Specs](https://github.com/microsoft/ProjectReunion/tree/master/specs) -> [Code](https://github.com/microsoft/ProjectReunion/tree/master/dev)
+  
+## 2022 and Beyond
+  
+In the future you can expect multiple releases per year from the Project Reunion team. We will continue expanding existing app compatibility and have a continuous delivery of new easier to use capabilities.
```

Comment:
> of new easier to use capabilities.

Nit: I'd drop "easier to use", that's subjective and is a mouth full here.

Reply:
Agreed that it's a mouth full here, I'd drop it for this sentence. Or restructure the entire sentence to say

> In the future you can expect multiple releases per year from the Project Reunion team. We will continue focusing on solving actual problems and pains Windows app developers have, with solutions that make Windows app development substantially easier.

Reply:
I think I dislike that more 😂 but of course this is a minor nit.

My keep-it-simple™ proposal:
> In the future, you can expect multiple releases per year from the Project Reunion team. We will continue to improve app compatibility and deliver new capabilities.

Reply:
I'm good with Rafael's simple sentence! The why and purpose of our work can be elsewhere, keep the roadmap simple